messaging-client-databus/src/main/java/com/inmobi/messaging/consumer/databus/AbstractMessagingDatabusConsumer.java

```diff
@@ -299,6 +295,15 @@ protected String getReaderNameSuffix() {
 
   protected abstract void createPartitionReaders() throws IOException;
 
+  public synchronized Long getPendingData() throws Exception {
```

Comment:
renaming to getPendingDataSize() done
Also added getPendingDataSize() to the interface. in the abstract consumer, the method will return -1 indicating the feature is not supported. consumers who has a concrete implementation of this method are expected to override getPendingDataSize() in their implementations